.github/workflows/ci.yml

```
@@ -32,57 +32,35 @@ jobs:
         shell: pwsh
         run: exit $(If ($env:SECRETS_AVAILABLE -eq 'true') { 0 } Else { 1 })
       - name: Checkout
-        if: github.event_name != 'pull_request_target'
-        uses: actions/checkout@v2.3.4
-        with:
-          fetch-depth: 0
-      - name: Checkout for Dependabot
-        if: github.event_name == 'pull_request_target'
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3
```

Should have told you earlier. We prefer to use fully-qualified versions so that we get a more deterministic build. Dependabot will then raise PRs to keep them up-to-date.
```suggestion
        uses: actions/checkout@v3.0.2
```